ecosystem/sep-0030.md

```diff
@@ -59,8 +59,13 @@ The signing flow is as follows:
 - The client, authenticated as the external authentication method (e.g. phone, email, etc) or another account, requests the server to sign a transaction.
 - The server verifies that the [JWT] is valid according to the authentication provider's specification.
 - The server verifies that the external authentication method (e.g. phone, email, etc) or account in the [JWT] is an alternative identity of the account.
-- The server verifies that the source account of the transaction is the account.
-- The server verifies that the source account of all operations on the transaction are not set or set to the account.
+- The server verifies that the source account of the transaction is one of:
```

Comment:
```suggestion
- The server verifies that the source account of the transaction is:
```

Reply:
I think we need the one of here otherwise it isn't explicit if the list is `and` or `or`. Is this incorrect to have one of here?

Reply:
You're right, and it is a small change anyway